demo/encouragementBotCommsWavyDot/vat-bot.js

```diff
@@ -13,7 +13,7 @@ export default function setup(syscall, state, helpers) {
       harden({
         encourageMe(nameObj) {
           // Test of tildot property get.
-          return nameObj~.name.then(name => {
+          return nameObj~.name~.identity().then(name => {
```

Comment:
The old code was exercising our ability to do `~.propname` GETs on a local object (the `{name: 'user'}` passed into `encourageMe()` would trigger the pass-by-copy behavior of marshal.serialize). This new code is not exercising that, but only GET on remote Presences.. should we find a way to retain that check?

Reply:
I'll revert this, as it isn't even testing the case I wanted to.